server.js

```diff
@@ -13,7 +13,7 @@ const PORT = process.env.PORT || 3001;
 app.use(express.json());
 app.use(cors());
 app.post('/user/register', async (req, res) => {
```

Comment:
The information to add to the database should be arriving from the front end with a post request